### DISCUSSION TITLE
Who discover Li?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#724] [Galaxy Trivia] [answered]
* Where?
* Li relation with 'V1369 Centauri'

**Comment by peterbaccaro** [comment, marked as answer]
The chemical element lithium (Li) was first discovered in **1817** by Swedish chemist **Johan August Arfwedson** while analysing petalite ore from the island of Utö in Sweden. He noticed an unknown alkali metal in the ore, which he named "lithium" after the Greek word "lithos" meaning stone.

As for V1369 Centauri, it is a binary star system located in the constellation Centaurus. In 2013, a nova occurred in the system, causing it to briefly brighten by a factor of 10,000. The nova was observed by several telescopes, including the Chandra X-ray Observatory, which detected the **presence of lithium in the nova's debris**.

The discovery of lithium in the debris of V1369 Centauri's nova was significant because it **provided evidence that lithium can be produced in nova explosions**. Prior to this discovery, it was thought that lithium could only be produced in the early universe or in the cores of stars.
